lib/mastodon/cli/maintenance.rb

```diff
@@ -604,11 +610,7 @@ def deduplicate_local_accounts!(accounts)
 
       say 'Please chose the one to keep unchanged, other ones will be automatically renamed.'
 
-      ref_id = ask('Account to keep unchanged:') do |q|
-        q.required true
-        q.default 0
-        q.convert :int
```

Comment:
This usage which had syntax as though it converted to an int, but thor doesn't actually have this feature -- I suspect this was missed during a previous commit which consolidated some other CLI input approaches to all use the thor helpers. Fixed this with explicit `to_i` conversion.